app/lib/methods/getSlashCommands.js

```diff
@@ -0,0 +1,32 @@
+import { InteractionManager } from 'react-native';
+
+import database from '../realm';
+import log from '../../utils/log';
+
+export default async function() {
+	try {
+		// RC 0.70.0
```

Comment:
Where did you get this 0.70.0 from?
It's `0.60.2` according to https://rocket.chat/docs/developer-guides/rest-api/commands/list/